playbooks/common/openshift-cluster/config.yml

```diff
@@ -18,6 +18,12 @@
       - docker_image_availability
       - docker_storage
 
+- hosts: localhost
+  tasks:
+  - fail:
+      msg: No etcd hosts defined. Running an all-in-one master is deprecated and will no longer be supported in a future upgrade.
+    when: groups.oo_etcd_to_config | default([]) | length == 0 and not openshift_master_unsupported_all_in_one | default(False)
```

Comment:
Please put `AND`d `when` statements on multiple lines in a list format, e.g. `groups.oo_etcd_to_config | default([]) | length == 0 and not openshift_master_unsupported_all_in_one | default(False)` should be more like this:

```yaml
    when:
      - groups.oo_etcd_to_config | default([]) | length == 0
      - not openshift_master_unsupported_all_in_one | default(False)
```